cpp/Ice/greeter/README.md

```diff
@@ -1,4 +1,6 @@
-This demo shows how to call and implement a canonical Greeter application using Ice.
+# Greeter
+
+The Greeter demo illustrates how to send a request and wait for the response.
```

Comment:
I think 'this' was more clear. Saying 'the greeter demo' sounds like you're talking about some other demo,
but we're talking about _this_ demo.

If you agree, here's a thing to save you some time:
```suggestion
This Greeter demo illustrates how to send a request and wait for the response.
```

Reply:
I prefer "The Greeter demo".  "This demo" when there is no qualification.